packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -302,9 +302,9 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
     } else {
       headerWidgets.add(new Expanded(child: new Text('$_selectedRowCount items selected')));
     }
-    if (config.actions != null) {
+    if (widget.actions != null) {
       headerWidgets.addAll(
-        config.actions.map<Widget>((Widget widget) {
+        widget.actions.map<Widget>((Widget widget) {
```

Comment:
This code now has two `widget` variables in scope, which is confusing. We should rename the inner one (maybe to "action").